### ISSUE TITLE
The newest update less than 50 seconds ago does not work. it will not reply with anything but I cannot do that.

### ISSUE THREAD

**Issue by getcheffy** [Batlez/ChatGPT-Jailbroken#3]
The newest update 
![no jailbreak](https://github.com/Batlez/ChatGPT-Jailbroken/assets/83881381/f49d5c27-7479-4106-a9be-d8dba0a81c0c)
less than 50 seconds ago does not work. it will not reply with anything but I cannot do that.

**Comment by Roman-Voloshin** [comment]
same

**Comment by walish88** [comment]
i am too :\

**Comment by Batlez** [comment]
Yeah it seems like OpenAI has started to patch these. I will continue to find a way to bypass this
